addons/mail/static/src/messaging/entity/attachment/attachment_tests.js

```diff
@@ -0,0 +1,154 @@
+odoo.define('mail.messaging.entity.AttachmentTests', function (require) {
+'use strict';
+
+const {
+    afterEach: utilsAfterEach,
+    beforeEach: utilsBeforeEach,
+    pause,
+    start: utilsStart,
+} = require('mail.messaging.testUtils');
+
+QUnit.module('mail', {}, function () {
+QUnit.module('messaging', {}, function () {
+QUnit.module('entity', {}, function () {
+QUnit.module('Attachment', {
+    beforeEach() {
+        utilsBeforeEach(this);
+        this.start = async params => {
+            if (this.widget) {
+                this.widget.destroy();
+            }
+            let { env, widget } = await utilsStart(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
+    },
+    afterEach() {
+        utilsAfterEach(this);
+        this.env = undefined;
+        if (this.widget) {
+            this.widget.destroy();
+            this.widget = undefined;
+        }
```

Comment:
do we need that in entity tests?

Reply:
should not be needed, but `start()` does mount some components like discuss or messaging menu.
We should later clean start so that it just setups the env